source/controller/wizard_controller.py

```diff
@@ -224,14 +223,22 @@ def set_default_project_location(self):
         self.wizard_window.next_button.setToolTip('Invalid Project Name')
 
     def set_equations_tabs(self):
+        n = len(dic_num_inv_spectral_names)
+        dic_num_inv_spectral_names1 = dict(list(dic_num_inv_spectral_names.items())[0: 27])
+        dic_num_inv_spectral_names2 = dict(list(dic_num_inv_spectral_names.items())[27: n])
```

Comment:
Why not take advantage of the value of n to make the matrix range dynamic instead of a fixed range?
 - Example:

```
dict(list(dic_num_inv_spectral_names.items())[0: n/2])
dict(list(dic_num_inv_spectral_names.items())[n/2: n])
```

Reply:
You need also round the number to avoid float numbers.

Reply:
Yes. At first I did it that way. But two very similar invariants would be in separate tabs: **_Smallest Eigen-A_** and **_Smallest Eigen-E_**, for example. The position 26 does the separation more consistently. Understand?